src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -252,9 +252,14 @@ protected async Task VerifyCustomCommitProviderAsync(string markupBeforeCommit,
             }
         }
 
-        protected virtual bool CompareItems(string actualItem, string expectedItem)
+        protected bool CompareItems(string actualItem, string expectedItem)
         {
-            return actualItem.Equals(expectedItem);
+            return GetStringComparer().Equals(actualItem, expectedItem);
+        }
+
+        protected virtual IEqualityComparer<string> GetStringComparer()
+        {
+            return StringComparer.Ordinal;
```

Comment:
nit: Should this be `StringComparer.Default` to be _exactly_ the same?

Reply:
There's no StringComparer.Defualt. If you're thinking of EqualityComparer<string>.Default, that'll just call into Equals but I'm preferring to be explicit of the intent here.